### ISSUE TITLE
Repo upload file error message

### ISSUE THREAD

**Issue by ghost** [go-gitea/gitea#8103]
- Gitea version (or commit ref): 1.9.0
- Git version: N/A
- Operating system: Windows
- Database:
  - [ ] PostgreSQL
  - [ ] MySQL
  - [x] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No (maybe due to different version, 1.10.0+dev-254-g5fcef38f7), but I can at https://git.teknik.io/Teknikode/Teknik/issues/109
  - [ ] Not relevant
- Log gist: N/A

## Description

After Gitea's 1.9.0 update, uploading files on a repo, entering a commit message and committing changes, returns the error message below. **However**, by pushing "Cancel" to return to the file listing, we can verify that the file(s) were successfully uploaded and committed.

```txt
Failed to upload files to '<path-to-current-dir>' with error: remove upload:
remove C:/Gitea/data/tmp/uploads/8/6/86ac3d46-7006-4a6e-a9e3-6f197afb27e5:
The process cannot access the file because it is being used by another process.
```

## Screenshots

![gitea-190-upload-error](https://user-images.githubusercontent.com/9649385/64361035-a304f080-d003-11e9-875f-0ab098054676.png)

**Comment by hzjianglf** [comment]
Today I install gitea fisrt time, met this problem too.

**Comment by megadr01d** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs during the next 2 weeks. Thank you for your contributions.

Duplicate of https://github.com/go-gitea/gitea/issues/8176#issuecomment-553229469.

**Comment by megadr01d** [comment]
This looks fixed on Gitea 1.10.3. Closing.

**Comment by HowardWhile** [comment]
I am met this problem too.
![image](https://user-images.githubusercontent.com/18715360/83709729-b2c46a80-a651-11ea-8d2c-6d55cf3ca090.png)

- Git version: 2.26.2,
- Operating system: Windows10
- Database:
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite

This is the gitea Version I tried
[v1.10.6](https://github.com/go-gitea/gitea/releases/tag/v1.10.6)
[v1.11.6](https://github.com/go-gitea/gitea/releases/tag/v1.11.6)
[v1.12.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.12.0-rc1)
